src/main/java/edu/harvard/h2ms/web/controller/ReportController.java

```diff
@@ -0,0 +1,33 @@
+package edu.harvard.h2ms.web.controller;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.harvard.h2ms.service.ReportService;
+
+@RestController
+@RequestMapping(path="/api/reports")

```

Comment:
Why `/api`? This makes it different than everything else, if we want a prefix like that, we can configure it at the global level (and we probably should)

Reply:
The other thoughts is why not just put this in the events controller at `/events/export` or something?

Reply:
This makes sense.  Changing to /events/export/csv